src/core.cpp

```diff
@@ -72,6 +81,10 @@ namespace Core
 	
 	/** The Address for the Last Round. **/
 	std::string LAST_ROUND_BLOCKFINDER = "2S4PPSznWfPVLtPJNpi8Ly46Wft3wGbayGkhaGzKLVcepmrhKTP";
+
+	unsigned int nConnections = 0;
+	unsigned int nLastBlockFound = 0;
```

Comment:
Oh, just noticed that in your newest changes the statscollector.cpp lines are removed. In this case it seems like nLastBlockFound is not used at all now.